tests/rl/test_helper_rl.py

```diff
@@ -0,0 +1,96 @@
+import os
+from pathlib import Path
+
+import pytest
+from mqt.bench import get_benchmark
+from qiskit import QuantumCircuit
+
+from mqt.predictor import rl
+
+
+def test_get_actions_opt():
+    assert len(rl.helper.get_actions_opt()) == 12
+
+
+def test_get_actions_layout():
+    assert len(rl.helper.get_actions_layout()) == 3
+
+
+def test_et_actions_routing():
+    assert len(rl.helper.get_actions_routing()) == 4
+
+
+def test_get_actions_platform_selection():
+    assert len(rl.helper.get_actions_platform_selection()) == 4
+
+
+def test_get_actions_synthesis():
+    assert len(rl.helper.get_actions_synthesis()) == 1
+
+
+def test_get_action_terminate():
+    assert len(rl.helper.get_action_terminate()) == 1
+
+
+def test_get_actions_devices():
+    assert len(rl.helper.get_actions_devices()) == 5
+
+
+@pytest.mark.skipif(
+    os.getenv("skip_optional_tests") == "true", reason="QASM files are not provided."
+)
```

Comment:
I might be wrong, but haven't you added a zip file with circuits in this PR? i.e., is this skip really necessary?

Reply:
True. I removed that `skipif` statement. Thanks for pointing it out.